### ISSUE TITLE
Unifi 2.4.27 upgrade

### ISSUE THREAD

**Issue by david00769** [peacey/split-vpn#168]
No issue - just fyi. My unifi OS auto upgraded to the new architecture last night (from the 1.x code base). The split vpn changes are all removed, and you'll need to reinstall the scripts if suddenly you find things aren't working.

**Comment by david00769** [comment]
also buried in the release notes is that there's a new SSH key to telnet in.

**Comment by peacey** [comment]
The release notes for 2.4 explicitly say that any custom modifications will be wiped because of the architecture change, so I hope you guys read the release notes, and make sure to backup your split-vpn configuration.

**Comment by david00769** [comment]
Yes. Saw that in the release notes, and took only about 2 minutes to get it up and running again.Sent from my iPadOn Feb 24, 2023, at 6:49 AM, peacey ***@***.***> wrote:﻿
The release notes for 2.4 explicitly say that any custom modifications will be wiped because of the architecture change, so I hope you guys read the release notes, and make sure to backup your split-vpn configuration.
